driver-core/src/main/com/mongodb/internal/connection/ClientMetadataHelper.java

```diff
@@ -37,63 +41,29 @@
  * <p>This class is not part of the public API and may be removed or changed at any time</p>
  */
 public final class ClientMetadataHelper {
-    public static final BsonDocument CLIENT_METADATA_DOCUMENT = new BsonDocument();
```

Comment:
Right. It's built via `getDriverInformation(null);` I suppose that this could be extracted back into this static, if there is no cache. (I'd like there to be a cache, but it seems to be breaking tests, as I mentioned elsewhere)